files/pt-br/web/javascript/guide/working_with_objects/index.html

```diff
@@ -78,7 +78,7 @@ <h2 id="Objetos_e_propriedades">Objetos e propriedades</h2>
 </pre>
 </div>
 
-<p>Você pode usar a notação de colchetes com o comando <a class="internal" href="/en-US/docs/JavaScript/Guide/Statements#for...in_Statement" title="en-US/docs/JavaScript/Guide/Statements#for...in Statement">for...in</a> para iterar por todas as propriedades enumeráveis de um objeto. Para ilustrar como isso funciona, a seguinte função mostra as propriedades de um objeto quando você passa o objeto e o nome do objeto como argumentos para a função:</p>
+<p>Você pode usar a notação de colchetes com o comando <a class="internal" href="en-US/docs/Web/JavaScript/Reference/Statements/for...in" title="en-US/docs/Web/JavaScript/Reference/Statements/for...in">for...in</a> para iterar por todas as propriedades enumeráveis de um objeto. Para ilustrar como isso funciona, a seguinte função mostra as propriedades de um objeto quando você passa o objeto e o nome do objeto como argumentos para a função:</p>
```

Comment:
Hi @LeoSanGo,
Thank you for your contribution.

I was wondering if you could change the link to portuguese version (`en-US/docs/Web/JavaScript/Reference/Statements/for...in` to `/pt-BR/docs/Web/JavaScript/Reference/Statements/for...in`). What do you think?

Reply:
Hi @josielrocha,
Good advice. I'll do it
Thank you!

Reply:
There is one more thing..
I think if we remove the `/` from the begining of the link we will end up introducing an invalid link. The correct link address should be `/pt-BR/docs/Web/JavaScript/Reference/Statements/for...in`, starting with /.

Would you do this little change?

Reply:
Exactly, done!
Thank you